### PR TITLE
ANON-201 - Rename http responses and some logs leftovers, remove links

### PR DESCRIPTION
**Pull request by yumirkov**

### CODE REVIEW

File: contrib/operator-tools/anon-exit-notice.html [hunk 1]
```diff
@@ -191,22 +191,22 @@ <h1>This is a Tor Exit Router</h1>
 <p>
 You are most likely accessing this website because you've had some issue with
 the traffic coming from this IP. This router is part of the <a
-href="https://www.torproject.org/">Tor Anonymity Network</a>, which is
+href="https://www.torproject.org/">Anon Anonymity Network</a>, which is
```
Comment (rA3ka) [inline contrib/operator-tools/anon-exit-notice.html]: Change to "Anyone Network"

File: contrib/operator-tools/anon-exit-notice.html [hunk 2]
```diff
@@ -191,22 +191,22 @@ <h1>This is a Tor Exit Router</h1>
...
 dedicated to <a href="https://2019.www.torproject.org/about/overview">providing
 privacy</a> to people who need it most: average computer users. This
 router IP should be generating no other traffic, unless it has been
 compromised.</p>
 
 <p>
-Tor works by running user traffic through a random chain of encrypted
-servers, and then letting the traffic exit the Tor network through an
+Anon works by running user traffic through a random chain of encrypted
+servers, and then letting the traffic exit the Anon network through an
```
Comment (rA3ka) [inline contrib/operator-tools/anon-exit-notice.html]: Anyone Network

File: contrib/operator-tools/anon-exit-notice.html [hunk 3]
```diff
@@ -191,22 +191,22 @@ <h1>This is a Tor Exit Router</h1>
...
 exit node like this one. This design makes it very hard for a service to
 know which user is connecting to it, since it can only see the IP-address
-of the Tor exit node:</p>
+of the Anon exit node:</p>
 
 <p style="text-align:center;margin:40px 0">
 <svg xmlns="http://www.w3.org/2000/svg" width="500" viewBox="0 0 490.28 293.73" style="width:100%;max-width:600px">
-<desc>Illustration showing how a user might connect to a service through the Tor network. The user first sends their data through three daisy-chained encrypted Tor servers that exist on three different continents. Then the last Tor server in the chain connects to the target service over the normal internet.</desc>
+<desc>Illustration showing how a user might connect to a service through the Anon network. The user first sends their data through three daisy-chained encrypted Anon servers that exist on three different continents. Then the last Anon server in the chain connects to the target service over the normal internet.</desc>
```
Comment (rA3ka) [inline contrib/operator-tools/anon-exit-notice.html]: Anyone Network

File: contrib/operator-tools/anon-exit-notice.html [hunk 4]
```diff
@@ -247,14 +247,14 @@ <h1>This is a Tor Exit Router</h1>
 </p>
 
 <p>
-<a href="https://2019.www.torproject.org/about/overview">Read more about how Tor works.</a></p>
+<a href="https://2019.www.torproject.org/about/overview">Read more about how Anon works.</a></p>
```
Comment (rA3ka) [inline contrib/operator-tools/anon-exit-notice.html]: https://anyone.io

File: contrib/operator-tools/anon-exit-notice.html [hunk 5]
```diff
@@ -247,14 +247,14 @@ <h1>This is a Tor Exit Router</h1>
...
 
 <p>
-Tor sees use by <a href="https://2019.www.torproject.org/about/torusers">many
+Anon sees use by <a href="https://2019.www.torproject.org/about/torusers">many
```
Comment (rA3ka) [inline contrib/operator-tools/anon-exit-notice.html]: https://anyone.io

File: contrib/operator-tools/anon-exit-notice.html [hunk 6]
```diff
@@ -247,14 +247,14 @@ <h1>This is a Tor Exit Router</h1>
...
 important segments of the population</a>, including whistle blowers,
 journalists, Chinese dissidents skirting the Great Firewall and oppressive
 censorship, abuse victims, stalker targets, the US military, and law
-enforcement, just to name a few.  While Tor is not designed for malicious
+enforcement, just to name a few.  While Anon is not designed for malicious
 computer users, it is true that they can use the network for malicious ends.
 In reality however, the actual amount of <a
 href="https://support.torproject.org/abuse/">abuse</a> is quite low. This
```
Comment (rA3ka) [inline contrib/operator-tools/anon-exit-notice.html]: https://anyone.io

File: contrib/operator-tools/anon-exit-notice.html [hunk 7]
```diff
@@ -264,23 +264,23 @@ <h1>This is a Tor Exit Router</h1>
 href="https://web.archive.org/web/20200131013910/http://voices.washingtonpost.com/securityfix/2008/08/web_fraud_20_tools.html">build,
```
Comment (rA3ka) [inline contrib/operator-tools/anon-exit-notice.html]: Delete URL

File: contrib/operator-tools/anon-exit-notice.html [hunk 8]
```diff
@@ -264,23 +264,23 @@ <h1>This is a Tor Exit Router</h1>
...
 sell, and trade</a> far larger and <a
 href="https://web.archive.org/web/20200131013908/http://voices.washingtonpost.com/securityfix/2008/08/web_fraud_20_distributing_your.html">more
```
Comment (rA3ka) [inline contrib/operator-tools/anon-exit-notice.html]: Delete URL

File: contrib/operator-tools/anon-exit-notice.html [hunk 9]
```diff
@@ -317,9 +317,9 @@ <h1>This is a Tor Exit Router</h1>
 <p>For more information, please consult the following documentation:</p>
 
 <div class="links">
```
Comment (rA3ka) [inline contrib/operator-tools/anon-exit-notice.html:313]: Remove from here

File: contrib/operator-tools/anon-exit-notice.html [hunk 10]
```diff
@@ -317,9 +317,9 @@ <h1>This is a Tor Exit Router</h1>
...
-<a href="https://2019.www.torproject.org/about/overview">Tor Overview</a>
-<a href="https://support.torproject.org/abuse/">Tor Abuse FAQ</a>
-<a href="https://community.torproject.org/relay/community-resources/eff-tor-legal-faq/">Tor Legal FAQ</a>
+<a href="https://2019.www.torproject.org/about/overview">Anon Overview</a>
+<a href="https://support.torproject.org/abuse/">Anon Abuse FAQ</a>
+<a href="https://community.torproject.org/relay/community-resources/eff-tor-legal-faq/">Anon Legal FAQ</a>
 </div>
```
Comment (rA3ka) [inline contrib/operator-tools/anon-exit-notice.html:317]: to here

File: contrib/operator-tools/anon-exit-notice.html [hunk 11]
```diff
@@ -191,22 +191,22 @@ <h1>This is a Tor Exit Router</h1>
 <p>
 You are most likely accessing this website because you've had some issue with
 the traffic coming from this IP. This router is part of the <a
-href="https://www.torproject.org/">Tor Anonymity Network</a>, which is
+href="https://www.torproject.org/">Anon Anonymity Network</a>, which is
 dedicated to <a href="https://2019.www.torproject.org/about/overview">providing
```
Comment (rA3ka) [inline contrib/operator-tools/anon-exit-notice.html]: https://anyone.io

File: contrib/operator-tools/anon-exit-notice.html [hunk 12]
```diff
@@ -247,14 +247,14 @@ <h1>This is a Tor Exit Router</h1>
 </p>
 
 <p>
-<a href="https://2019.www.torproject.org/about/overview">Read more about how Tor works.</a></p>
+<a href="https://2019.www.torproject.org/about/overview">Read more about how Anon works.</a></p>
 
 <p>
```
Comment (rA3ka) [inline contrib/operator-tools/anon-exit-notice.html:249]: Suggestion to remove everything from here to last line.

File: debian/control [hunk 13]
```diff
@@ -18,34 +18,34 @@ Breaks: ${runit:Breaks}
...
  downstream relay.
  .
- Basically, Tor provides a distributed network of relays. Users bounce
+ Basically, Anon provides a distributed network of relays. Users bounce
  their TCP streams (web traffic, ftp, ssh, etc) around the relays, and
  recipients, observers, and even the relays themselves have difficulty
  learning which users connected to which destinations.
  .
- This package enables only a Tor client by default, but it can also be
+ This package enables only a Anon client by default, but it can also be
```
Comment (rA3ka) [inline debian/control:34]: Anyone client

File: debian/control [hunk 14]
```diff
@@ -18,34 +18,34 @@ Breaks: ${runit:Breaks}
...
  configured as a relay and/or a hidden service easily.
  .
- Client applications can use the Tor network by connecting to the local
- socks proxy interface provided by your Tor instance. If the application
+ Client applications can use the Anon network by connecting to the local
```
Comment (rA3ka) [inline debian/control]: Anyone Network

File: debian/control [hunk 15]
```diff
@@ -18,34 +18,34 @@ Breaks: ${runit:Breaks}
...
+ socks proxy interface provided by your Anon instance. If the application
```
Comment (rA3ka) [inline debian/control:38]: Remove from "If the application..."

File: debian/control [hunk 16]
```diff
@@ -18,34 +18,34 @@ Breaks: ${runit:Breaks}
...
  itself does not come with socks support, you can use a socks client
  such as torsocks.
```
Comment (rA3ka) [inline debian/control:40]: To end of this line.

File: debian/control [hunk 17]
```diff
@@ -18,34 +18,34 @@ Breaks: ${runit:Breaks}
...
  .
- Note that Tor does no protocol cleaning on application traffic. There
+ Note that Anon does no protocol cleaning on application traffic. There
  is a danger that application protocols and associated programs can be
- induced to reveal information about the user. Tor depends on Torbutton
```
Comment (rA3ka) [inline debian/control:44]: Remove from "Tor depends on .."

File: debian/control [hunk 18]
```diff
@@ -18,34 +18,34 @@ Breaks: ${runit:Breaks}
...
+ induced to reveal information about the user. Anon depends on Torbutton
  and similar protocol cleaners to solve this problem. For best
- protection when web surfing, the Tor Project recommends that you use
- the Tor Browser Bundle, a standalone tarball that includes static
- builds of Tor, Torbutton, and a modified Firefox that is patched to fix
+ protection when web surfing, the Anon Project recommends that you use
+ the Anon Browser Bundle, a standalone tarball that includes static
+ builds of Anon, Torbutton, and a modified Firefox that is patched to fix
  a variety of privacy bugs.
```
Comment (rA3ka) [inline debian/control:49]: To end of this line 49.

File: debian/control [hunk 19]
```diff
@@ -54,15 +54,15 @@ Priority: optional
...
- This package provides a GeoIP database for Tor, i.e. it maps IPv4 addresses
+Description: GeoIP database for Anon
+ This package provides a GeoIP database for Anon, i.e. it maps IPv4 addresses
  to countries.
  .
- Bridge relays (special Tor relays that aren't listed in the main Tor
+ Bridge relays (special Anon relays that aren't listed in the main Anon
  directory) use this information to report which countries they see
- connections from.  These statistics enable the Tor network operators to
+ connections from.  These statistics enable the Anon network operators to
```
Comment (rA3ka) [inline debian/control]: Anyone Network

File: debian/control [hunk 20]
```diff
@@ -54,15 +54,15 @@ Priority: optional
...
  learn when certain countries start blocking access to bridges.
  .
  Clients can also use this to learn what country each relay is in, so
- Tor controllers like arm or Vidalia can use it, or if they want to
+ Anon controllers like arm or Vidalia can use it, or if they want to
```
Comment (rA3ka) [inline debian/control:67]: Remove line 67-68.

File: scripts/maint/geoip/geoip-db-tool/src/main.rs [hunk 21]
```diff
@@ -122,8 +122,8 @@ impl NetDefn {
 
 const PROLOGUE: &str = "\
 # This file has been converted from the IPFire Location database
-# using Tor's geoip-db-tool, which is available in the
-# scripts/maint/geoip/geoip-db-tool directory in the Tor source
+# using Anon's geoip-db-tool, which is available in the
+# scripts/maint/geoip/geoip-db-tool directory in the Anon source
 # code repository at https://gitlab.torproject.org/tpo/core/tor/ .
```
Comment (rA3ka) [inline scripts/maint/geoip/geoip-db-tool/src/main.rs]: Link to anyone repo instead.

**Review by rA3ka — COMMENTED** [review]
Commented on changes

File: contrib/operator-tools/anon-exit-notice.html [hunk 22]
```diff
@@ -10,8 +10,7 @@
 This notice is intended to be placed on a virtual host for a domain that
 your Tor exit node IP reverse resolves to so that people who may be about
```
Comment (rA3ka) [inline contrib/operator-tools/anon-exit-notice.html:11]: Anon exit

File: contrib/operator-tools/anon-exit-notice.html
Comment (0xneuratic) [inline contrib/operator-tools/anon-exit-notice.html:1]: I would remove 329 onwards, not sure if we will provide such a database of exit IPs

File: contrib/operator-tools/anon-exit-notice.html [hunk 23]
```diff
@@ -2,16 +2,15 @@
 <html lang="en-US">
 <head>
 <meta charset="UTF-8">
-<title>This is a Tor Exit Router</title>
+<title>This is a Anon Exit Router</title>
```
Comment (kanshi) [inline contrib/operator-tools/anon-exit-notice.html:5]: Should be This is an Anon Exit Router

File: contrib/operator-tools/anon-exit-notice.html [hunk 24]
```diff
@@ -79,7 +78,7 @@
 </head>
 <body>
 <main>
-<h1>This is a Tor Exit Router</h1>
+<h1>This is a Anon Exit Router</h1>
```
Comment (kanshi) [inline contrib/operator-tools/anon-exit-notice.html:81]: This should be 'an Anon ...'

**Review by kanshi — APPROVED** [review]

**Pull request merged**